src/pyskindose/geom_calc.py

```diff
@@ -417,13 +407,13 @@ def check_table_hits(source: np.array, table: Phantom, beam,
     # Create triangles:
 
     # Define edges of table (see illustration in project documentation)
-    a = table.r[6, :]
-    a1 = table.r[7, :]
-    a2 = table.r[5, :]
+    a = table.r[0 + 4, :]
```

Comment:
Is the change in indices for this and row 414 below intentional

Reply:
yes, it was changed to fit the new coordinate standard in e2725669128fd74dc0cb86a1ca14b1bccbc7d658